setup/jest.config.js

```diff
@@ -132,7 +132,6 @@ module.exports = {
     'src/modules/common/components/timestamp/index.js',
```

Comment:
I can still get duplicate network by clicking edit and then save
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/7c4af3e0-f287-4cb1-ba1e-f7a2ca91a569)